CorsixTH/Lua/languages/english.lua

```diff
@@ -448,14 +483,14 @@ tooltip.folders_window = {
   browse = "Browse for folder location",
   data_location = "The directory of the original Theme Hospital installation, which is required to run CorsixTH",
   font_location = "Location of a font file that is capable of displaying Unicode characters required by your language. If this is not specified you will not be able to choose languages that need more characters than the original game can supply. Example: Russian and Chinese",
-  savegames_location = "By default, the Saves directory is alongside the config file and will be used for storing saved games in. Should this not be suitable, then you can choose your own, just browse to the directory which you want to use.",
-  screenshots_location = "By default, the Screenshots are stored in a folder alongside the config file. Should this not be suitable, then you can choose your own, just browse to the directory which you want to use.",
-  music_location = "Select a location for your music files.  You must have created the directory already, then browse to the directory which you just created.",
+  savegames_location = "By default, the Saves directory is alongside the config file and will be used for storing saved games in. Alternatively, you can choose your own. Just browse to the directory that you want to use.",
```

Comment:
If we are fixing this prompt anyway - I don't like the tone of the last sentence. Perhaps `Alternatively, you can choose your own by browsing to the directory that you would like to use.`